two_factor/templates/two_factor/email/email_template.html

```diff
@@ -0,0 +1 @@
+Your authentication token for user {{ user }} is {{ token }}.
```

Comment:
* This should be in `{% blocktrans %}` tags at least.
* It's a bit curt and doesn't say where the email has come from (imagine the user received this message unexpectedly due to a compromised password)